hikari/api/rest.py

```diff
@@ -7820,6 +7845,34 @@ async def create_autocomplete_response(
             If an internal error occurs on Discord while handling the request.
         """  # noqa: E501 - Line too long
 
+    async def create_modal_response(
+        self,
+        interaction: snowflakes.SnowflakeishOr[base_interactions.PartialInteraction],
+        token: str,
+        *,
+        title: str,
+        custom_id: str,
+        components: typing.Sequence[special_endpoints.ComponentBuilder],
+    ) -> None:
+        """Create a response by sending a modal.
+
+        Parameters
+        ----------
+        interaction : hikari.snowflakes.SnowflakeishOr[hikari.interactions.base_interactions.PartialInteraction]
+            Object or ID of the interaction this response is for.
+        token : builtins.str
+            The command interaction's token.
```

Comment:
Should these be documented?

They don't appear in the type hints as arguments you can pass to the method.
Signature: `(method) create_modal_response: (title: str, custom_id: str, components: Sequence[ComponentBuilder]) -> Coroutine[Any, Any, None]`

Reply:
what?

Reply:
the docstring does not match the type hint in terms of parameters.

Reply:
ah I see what you're talking about. You commented on the wrong line.